org.lflang/src/org/lflang/federated/validation/FedValidator.java

```diff
@@ -0,0 +1,69 @@
+package org.lflang.federated.validation;
```

Comment:
Wouldn't this be better located in `org.lflang.validation`?

Reply:
I agree, but there a few examples of badly placed classes. I've created an issue so we can move on and do this later: https://github.com/lf-lang/lingua-franca/issues/1561